packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -39,10 +43,14 @@ class CodeView extends StatefulWidget {
   final void Function(ScriptRef scriptRef, int line) onSelected;
 
   @override
-  _CodeViewState createState() => _CodeViewState();
+  _CodeViewState createState() => _CodeViewState(controller);
```

Comment:
https://dart-lang.github.io/linter/lints/no_logic_in_create_state.html
will avoid this issue without code review. lets add the lint to devtools.